tests/offline_mode/test_textfilecontent54.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
+
+# Copyright 2018 Red Hat Inc., Durham, North Carolina.
+# All Rights Reserved.
+#
+# OpenSCAP Test Suite
+#
+# Authors:
+#      Jan Černý <jcerny@redhat.com>
+
+. ${srcdir}/../test_common.sh
+
+set -e -o pipefail
+
+function test_textfilecontent54 {
+    temp_dir="$(mktemp -d)"
+
+    # prepare /bar.txt
+    echo "Hello" > "$temp_dir/bar.txt"
+
+    # prepare /zzz/foo.txt
+    mkdir -p "$temp_dir/zzz"
+    echo "Bye" > "$temp_dir/zzz/foo.txt"
+
+    result="$(mktemp)"
+
+    export OSCAP_PROBE_ROOT
+    OSCAP_PROBE_ROOT="$temp_dir"
+    $OSCAP oval eval --results $result $srcdir/textfilecontent54.oval.xml
+
+    [ -s "$result" ]
+
+    assert_exists 1 '/oval_results/results/system/oval_system_characteristics/system_data'
+
+    tfc_item='/oval_results/results/system/oval_system_characteristics/system_data/ind-sys:textfilecontent_item'
+    assert_exists 2 $tfc_item
+
+    assert_exists 1 $tfc_item'/ind-sys:filepath[text()="/bar.txt"]'
+    assert_exists 1 $tfc_item'/ind-sys:path[text()="/"]'
+    assert_exists 1 $tfc_item'/ind-sys:filename[text()="bar.txt"]'
+    assert_exists 1 $tfc_item'/ind-sys:text[text()="Hello"]'
+
+    assert_exists 1 $tfc_item'/ind-sys:filepath[text()="/zzz/foo.txt"]'
+    assert_exists 1 $tfc_item'/ind-sys:path[text()="/zzz"]'
+    assert_exists 1 $tfc_item'/ind-sys:filename[text()="foo.txt"]'
+    assert_exists 1 $tfc_item'/ind-sys:text[text()="Bye"]'
+
+    rm -rf "$temp_dir"
+    rm -f "$result"
+}
+
+# Testing.
+
+test_init "test_textfilecontent54.log"
+
+test_run "test_textfilecontent54" test_textfilecontent54
```

Comment:
this is not a good name for this test. it should have offline in the name, otherwise it's called exactly the same as another test that is not offline

Reply:
(try looking for this test in jenkins results)

Reply:
Actually I am pretty sure this test is never run, could you double check this?

Reply:
Nvm, I had a typo in my grep string:

```
Making check in offline_mode
make[3]: Entering directory `/home/jenkins/workspace/openscap-pull-requests/openscap-1.2.17/_build/tests/offline_mode'
make  check-TESTS
make[4]: Entering directory `/home/jenkins/workspace/openscap-pull-requests/openscap-1.2.17/_build/tests/offline_mode'

----------------------------------------------------------------------
+ test_system_info                                            [ PASS ]
--------------------------------------------------
See tests/offline_mode/test_system_info.log.

PASS: test_system_info.sh

----------------------------------------------------------------------
+ test_textfilecontent54                                      [ PASS ]
--------------------------------------------------
See tests/offline_mode/test_textfilecontent54.log.
```

I still wish both tests had better names.

Reply:
I checked and it is in Jenkins output.

I meant the `offline_mode` directory should contain only tests that are related only to offline mode.

But if you wish I will rename it.